ddtrace/internal/telemetry/constants.py

```diff
@@ -16,6 +16,13 @@ class TELEMETRY_LOG_LEVEL(Enum):
     ERROR = "ERROR"
 
 
+class TELEMETRY_APM_PRODUCT(Enum):
```

Comment:
Should Tracing not be included as a product?

Reply:
it think it should - when we started this project it was just assumed that tracing is used (because the only product was profiling), but nowadays it totally makes sense to track it individually. 

Since supporting it will require backend changes (same as the ones i've made for LLM Obs), I suggest leaving it to a future PR - I'll work with the new Telemetry EM to make sure there's a task for that.